src/Services/ListService.php

```diff
@@ -65,6 +65,17 @@ public function getListOptionsForLists($lists)
         return $records;
     }
 
+    public function getListIds()
+    {
+        $sql = "SELECT DISTINCT list_id FROM list_options ORDER BY list_id";
+        $records = QueryUtils::fetchRecords($sql, $lists, false);
```

Comment:
QueryUtils::fetchTableColumn($sql, 'list_id', []); will accomplish this for you as well, but you don't need to change this if you don't want to.